posydon/interpolation/profile_interpolation.py

```diff
@@ -149,13 +157,21 @@ def save(self, filename):
                    for key in SAVE_ATTRS if key not in DONT_SAVE}
         with open(filename, 'wb') as f:
             pickle.dump(myattrs, f)
-            
+
             
 class ProfileInterpolator:
     
-    def __init__(self):
+    def __init__(self,seed_value):
```

Comment:
I suggest to set `seed_value=None` as default, so you can call the function without specifying it.

Reply:
good idea. Done!